src/module/api/timeline/update/index.ts

```diff
@@ -25,10 +25,6 @@ export async function Update(
   obj.getMetadataMap().set(key.TimelineID, updateTimeline.id);
   obj.getMetadataMap().set(key.VentureID, updateTimeline.ventureId);
 
-  if (updateTimeline.visibility) {
-    obj.getMetadataMap().set(key.ResourceVisibility, updateTimeline.visibility);
-  }
-
```

Comment:
What is the different between this one and the one you changed to 
```
if (updateTimeline.visibility) {
     const patch = new UpdateI_Obj_Jsnpatch();
     patch.setOpe("replace");
     patch.setPat(`/obj/metadata/${key.ResourceVisibility.replace("/", "~1")}`);
     patch.setVal(updateTimeline.visibility);
     patchList.push(patch);
   }
```

Reply:
`obj.getMetadataMap()` sets the metadata for the API request (sort of like HTTP headers) while the JSON patch is a patch to be applied to the actual object in the database. These are operating at different levels of responsibility--GRPC layer vs business layer. Sometimes this is confusing though which is why I made this mistake :)